README.md

```diff
@@ -48,28 +52,40 @@ scrot requires a few projects and libraries:
 - libXfixes [(can be found in X.Org)](https://gitlab.freedesktop.org/xorg/lib/libxfixes)
 - libXinerama [(can be found in X.Org)](https://gitlab.freedesktop.org/xorg/lib/libxinerama)
 
-From a Unix command line, building is simple:
+### Generic installation instruction ###
+
+In short, if you are building from an GIT checkout of scrot (but not if you
+are building from a downloaded tarball) then you will first need to run:
 ```
-$ ./autogen.sh
-$ ./configure
-$ make
+./autogen.sh
 ```
 
-Cleaning up the build is also simple:
+TIP: if the source code does not have a configure file, you will need to run
+     the ```./autogen.sh``` command.
+
+Now, proceed as shown below for a tarball. (If you are using the tarball
+but have applied additional patches then you may also need to run
+```./autogen.sh```.)
+
+To build scrot from a downloaded tarball just type:
 ```
-$ make distclean
+./configure
+make
 ```
 
-You can return to a pristine source tree before running `./configure`:
+And as root:
 ```
-$ ./autogen.sh clean
+make install
 ```
 
-## Installing ##
+Cleaning up the build is also simple:
+```
+$ make distclean
```

Comment:
Note that `make clean` only cleans the build(`rm src/scrot; rm src/*.o`) but `distclean` removes all build + configure script: the user is forced to run `autogen.sh` in the latter case.

Reply:
Yes, right. What you think about:


Cleaning up the build is also simple:
```
make clean
```

To vanish the source code, removing all pre-built files, including ```configure```, run:
```
make distclean
```

Reply:
Yes, I agree

Reply:
> Yes, right. What you think about:
> 
> Cleaning up the build is also simple:
> 
> ```
> make clean
> ```
> 
> To vanish the source code, removing all pre-built files, including `configure`, run:
> 
> ```
> make distclean
> ```


My web browser went crazy, it doesn't publish some comments or they are duplicated xD
But just in case, to this your query, is `Yes, I agree`